src/languages/turtle.js

```diff
@@ -6,7 +6,7 @@ Description: Terse RDF Triple Language for the semantic web
 Website: https://www.w3.org/TR/turtle/
 */
 
-function(hljs) {
+module.exports = function(hljs) {
```

Comment:
Preferably if we're building with Highlight.js built tool this would all be ES6 modules (export keyword, not `module.exports), not CJS exports...  The build system currently supports both, but that may not always be the case. 

We'll likely still spit out "web" (`hljs` global) and ESM builds into `dist` for a while, I'm specifically talking about the source files in `src/languages` here.

Reply:
No worries. I had been looking at some other language extensions and think I was mostly seeing commonJS exports, but I can make the switch.

Reply:
Well of course this has implications if someone is going to use the package directly as well, but Highlight.js is going ESM only with v12... so I think that is where everything is headed for sure.  And many people already switched last year even.